Tests/SwiftSDKGeneratorTests/SwiftSDKRecipes/LinuxRecipeTests.swift

```diff
@@ -154,8 +154,37 @@ final class LinuxRecipeTests: XCTestCase {
     XCTAssertEqual(foundHostSwift, includesHostSwift)
   }
 
-  func testItemsToDownloadForMacOSHost() throws {
-    let hostTriple = Triple("x86_64-apple-macos")
+  func testItemsToDownloadForMacOSHost_noArchs() throws {
+    try testItemsToDownloadForMacOSHost(hostTriples: []) { result in
+      XCTAssertThrowsError(
+        try result.get(),
+        "",
+        { error in
+          if case DownloadableArtifacts.Error.noHostTriples = error {
+            return
+          }
+          XCTFail("unexpected error: \(error)")
+        }
+      )
+    }
+  }
+
+  func testItemsToDownloadForMacOSHost_arm64() throws {
+    try testItemsToDownloadForMacOSHost(hostTriples: [Triple("arm64-apple-macos")])
+  }
+
+  func testItemsToDownloadForMacOSHost_x86_64() throws {
+    try testItemsToDownloadForMacOSHost(hostTriples: [Triple("x86_64-apple-macos")])
+  }
+
+  func testItemsToDownloadForMacOSHost_multiArch() throws {
+    try testItemsToDownloadForMacOSHost(hostTriples: [Triple("arm64-apple-macos"), Triple("x86_64-apple-macos")])
+  }
+
+  private func testItemsToDownloadForMacOSHost(
+    hostTriples: [Triple],
+    validate: ((Result<Void, Error>) throws -> Void)? = nil
+  ) throws {
```

Comment:
It took me a minute to track down logic for `validate`, would you mind adding a doc comment here?

```suggestion
  /// ...
  /// - Parameters:
  ///   - hostTriples: ...
  ///   - validate: ...
  /// - Throws: ...
  private func testItemsToDownloadForMacOSHost(
    hostTriples: [Triple],
    validate: ((Result<Void, Error>) throws -> Void)? = nil
  ) throws {
```

Reply:
Done